packages/manager/modules/pci/src/projects/project/instances/instance/stop/stop.controller.js

```diff
@@ -0,0 +1,73 @@
+import get from 'lodash/get';
+
+export default class PciInstanceStopController {
+  /* @ngInject */
+  constructor(
+    $q,
+    $translate,
+    Poller,
+    CucCloudMessage,
+    PciProjectsProjectInstanceService,
+    OvhApiCloudProjectInstance,
+  ) {
+    this.$q = $q;
+    this.$translate = $translate;
+    this.Poller = Poller;
+    this.CucCloudMessage = CucCloudMessage;
+    this.PciProjectsProjectInstanceService = PciProjectsProjectInstanceService;
+    this.OvhApiCloudProjectInstance = OvhApiCloudProjectInstance;
+  }
+
+  $onInit() {
+    this.isLoading = false;
+  }
+
+  waitInstanceStop() {
+    const endPointUrl = `/cloud/project/${this.projectId}/instance/${this.instance.id}`;
+    return this.$q((resolve) => {
```

Comment:
Does `$q` needed? Why not simply return `this.Poller.poll`?

Reply:
Nice, i just forgot it i was used it for a test when i start.
So not need it anymore.